### PR TITLE
Theodorkl/LSP integration

### PR DESCRIPTION
**Pull request by theodorklauritzen**
- Add closing html tag to all opened html tags
- Change some function signatures to make the documentation more uniform and easier to parse.

### CODE REVIEW

File: en/reference/rank-features.html [hunk 1]
```diff
@@ -847,7 +847,7 @@ <h3 id="attribute-match-features-normalized">Attribute match features - normaliz
      </p><p>
      <code>max( 1-age(name)/maxAge , 0 )</code>
      </p><p>
-     Scales linearly with age, see <a href="#freshness">freshness plot</a>.
+     Scales linearly with age, see <a href="#freshness">freshness plot</a>.</p>
```
Comment (esolitos) [inline en/reference/rank-features.html:850]: This should be closed above, as `<p>` taggas cannot be nested.
> The [P](https://www.w3.org/TR/html401/struct/text.html#edef-P) element represents a paragraph. It cannot contain [block-level elements](https://www.w3.org/TR/html401/struct/global.html#block-inline) (including [P](https://www.w3.org/TR/html401/struct/text.html#edef-P) itself).
> 
> Source: https://www.w3.org/TR/html401/struct/text.html#h-9.3.1

**Review by esolitos — COMMENTED** [review]
ℹ️ The html tags `<p>` and `<li>` (among many others) don't need to be closed **in HTML** to be valid, this is only required for XHTML/XML. 😉  
_Ref 👉🏼 [Tag omission in text/html](https://html.spec.whatwg.org/multipage/dom.html#concept-element-tag-omission)_

Beside that, nice initiative! 👍🏼

**Comment by theodorklauritzen** [comment]
Thank I'll look into that!

File: en/reference/schema-reference.html [hunk 2]
```diff
@@ -1005,9 +1008,10 @@ <h2 id="field">field</h2>
             separately. Note that only URLs can be indexed this way, not other URIs.
             The different components are as defined by the HTTP standard:
             Scheme, hostname, port, path, query and fragment. Example:
-          <pre>
-http://mysite.mydomain.com:8080/path/shop?d=hab&amp;id=1804905709&amp;cat=100#frag1
-</pre>
+            <pre>
+              http://mysite.mydomain.com:8080/path/shop?d=hab&amp;id=1804905709&amp;cat=100#frag1
+            </pre>
```
Comment (kkraune) [inline en/reference/schema-reference.html:1013]: As the pre elements preserve whitespace, they look better without this indentation

```suggestion
<pre>
http://mysite.mydomain.com:8080/path/shop?d=hab&amp;id=1804905709&amp;cat=100#frag1
</pre>
```

**Review by kkraune — APPROVED** [review]

**Comment by theodorklauritzen** [comment]
I have started over in a new branch here #3348, and will close this PR

**Pull request closed without merging**